shared/properties-view/properties-view.hpp

```diff
@@ -143,6 +144,19 @@ class OBSPropertiesView : public VScrollArea {
 	void GetScrollPos(int &h, int &v, int &hend, int &vend);
 	void SetScrollPos(int h, int v, int old_hend, int old_vend);
 
+	using frame_rate_range_t =
+		std::pair<media_frames_per_second, media_frames_per_second>;
+	using frame_rate_ranges_t = std::vector<frame_rate_range_t>;
+
+	static QWidget *
+	CreateRationalFPS(OBSFrameRatePropertyWidget *fpsProps, bool &selected,
+			  const media_frames_per_second *current_fps);
+	static OBSFrameRatePropertyWidget *
+	CreateFrameRateWidget(obs_property_t *prop, bool &warning,
+			      const char *option,
+			      media_frames_per_second *current_fps,
+			      frame_rate_ranges_t &fps_ranges);
+
```

Comment:
Why is this addition needed?

Reply:
Mainly to access the `tr` function.

Reply:
Replace this by using `QObject::tr` in static functions.